dev/Common/CornerRadiusToFilteredMultipliedThicknessConverter.idl

```diff
@@ -0,0 +1,32 @@
+﻿namespace MU_XCP_NAMESPACE
+{
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+[default_interface]
+runtimeclass CornerRadiusToFilteredMultipliedThicknessConverter : Windows.UI.Xaml.DependencyObject, Windows.UI.Xaml.Data.IValueConverter
```

Comment:
Could we use CornerRadiusToThicknessConverter and add the Multiplier property?

Reply:
Unfortunately in it's current form, the CornerRadiusToThicknessConverter wouldn't help us with a multiplier property. The CornerRadiusToThicknessConverter always sets two directions to zero and the other two to the corner radius value. 
Essentially it would move the item diagonally. In our case however, we would need to move the item the left/right which the CornerRadiusToThicknessConverter can't provide.

Reply:
The existing CornerRadiusToThicknessConverter sets two components from the Thickness from the CornerRadius, and sets the rest to zero. The new converter needs to take _one_ component from the CornerRadius and set the rest to zero. So couldn't we reuse it with a new [CornerRadiusToThicknessConverterKind](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.Primitives.CornerRadiusToThicknessConverterKind) (plus the multipler)?

public enum CornerRadiusToThicknessConverterKind
{
    FilterTopAndBottomFromLeft = 0,
    FilterTopAndBottomFromRight = 1,
    FilterLeftAndRightFromTop = 2,
    FilterLeftAndRightFromBottom = 3,

    // new
    FilterLeftFromBottomLeft
    FilterTopFromTopLeft
    ...
}

Reply:
It seems like you could expand the CornerRadiusToThicknessConverterKind to have more enum values like FilterTopFromTopLeft and then also add the multiplier property

---
In reply to: [450386895](https://github.com/microsoft/microsoft-ui-xaml/pull/2827#discussion_r450386895) [](ancestors = 450386895)

Reply:
All right, I'll update the existing converter. So we would introduce 8 new values to the enum right?

LeftFromBottomLeft,
LeftFromTopLeft,
TopFromTopLeft,
TopFromTopRight,
and so on.

Reply:
Updated the existing converter.